doc/dev/kubernetes.rst

```diff
@@ -166,40 +171,33 @@ build, and the Ceph build too.
 5. Run a Rook cluster
 =====================
 
-.. note::
-
-    This is just some basic instructions: the Rook documentation
-    is much more expansive, at https://github.com/rook/rook/tree/master/Documentation
+Please refer to `Rook's documentation <https://github.com/rook/rook/tree/master/Documentation>`_
```

Comment:
I'd suggest using a link to the generated documentation: https://rook.io/docs/rook/master/

Reply:
A link to the quickstart guide may be better. https://rook.io/docs/rook/master/ceph-quickstart.html